app/src/drizzleOptions.js

```diff
@@ -0,0 +1,22 @@
+import SimpleStorage from "./contracts/SimpleStorage.json";
+import ComplexStorage from "./contracts/ComplexStorage.json";
+import TutorialToken from "./contracts/TutorialToken.json";
+
+const options = {
+  web3: {
+    block: false,
+    fallback: {
+      type: "ws",
+      url: "ws://127.0.0.1:9545",
```

Comment:
Ah cool, so recommending `truffle develop` as the starting tool

Reply:
Yup.